### ISSUE TITLE
ISO 22057 - unique identifiers

### ISSUE THREAD

**Issue by RobertSpang** [InDataWG/ILCD-EPD-Data-Format#27]
### I raise this change request because...

- [ ] the ILCD+EPD format lacks compliance with regualtions.
- [X] the ILCD+EPD format lacks conformaty with standards.
- [ ] the ILCD+EPD format needs to be improved regarding existing format or master data elements.
- [ ] there is another problem or use case I´m trying to address.

### Problem description

unique identifiers shall be listed.
Think there are two levels:
- product level (e.g. GTIN number)
- company level (e.g. VAT number)

### Solution description

Needs to be discussed and aligned with @martin erlandson

### Relevance or added value

high

**Comment by okworx** [comment]
Product IDs are already addressed in #36 .

VAT number to be added to contact dataset.

**Comment by okworx** [comment]
@ domain experts: please review and approve (by giving a thumbs up 👍) or else comment

**Comment by RobertSpang** [comment]
Repeating here; pls double check with screenshot. And these was already a comment were I posted it
![grafik](https://github.com/user-attachments/assets/5e853719-d21e-418f-a790-d7270fbdf273)

**Comment by okworx** [comment]
> Repeating here; pls double check with screenshot. And these was already a comment (on produc t identifiers if I´m not mistaken) were I posted it)

Yes this is covered, see above. Product IDs are already addressed in https://github.com/InDataWG/ILCD-EPD-Data-Format/issues/36 .

**Comment by RobertSpang** [comment]
But I don´t see operator and facility identifier in above list. So how are these implemented?

**Comment by RobertSpang** [comment]
> Product IDs are already addressed in #36 .
> 
> VAT number to be added to contact dataset, along with unique manufacturer ID(s). For that, we'll use a similar structure as for product IDs in order to include arbitrary company identifiers in a contact dataset:
> 
> ```
>                 <e:entityIds>
>                    <e:entityId e:type="VATID">DE213119920</e:entityId>
>                    <e:entityId e:type="EC_ORG_ID">2938547239487234987</e:entityId>
>                    <e:entityId e:type="TIN">9F5536320</e:entityId>
>                    <e:entityId e:type="domain">woodproductssa.it</e:entityId>
>                 </e:entityIds>
> ```
> 
> type can be `VATID` for VAT IDs `TIN` for Tax Identification Numbers `domain` for domain names as used in openEPD
> 
> If the EC comes up with some sort of other identifiers, this can then be used as well.

But I don´t see operator and facility identifier in above list. So how are these implemented?
